docker/grafana/setup-grafana

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env python
+
+import os
+import subprocess
+
+
+def psql_sql(sql):
+    p = subprocess.Popen(["psql", "-h", "postgres", "-U", "chroma", "-tAc {}".format(sql)], stdout=subprocess.PIPE)
+    return p.stdout.read()
+
+
+out = psql_sql("SELECT datname FROM pg_catalog.pg_database WHERE datname = 'grafana'")
+print("Does grafana database exist?: {}".format("grafana" in out))
+if "grafana" not in out:
+    create_db_out = psql_sql("CREATE DATABASE grafana OWNER {}".format(os.getenv("DB_USER")))
+    print("Database 'grafana' created: {}".format("CREATE DATABASE" in create_db_out))
```

Comment:
There is [a one-liner](https://stackoverflow.com/a/18389184) for "create if not exists" :)

Reply:
Nice, i'll update.